### ISSUE TITLE
bug(material/chips): Focus state for input chips has insufficient text contrast

### ISSUE THREAD

**Issue by zarend** [angular/components#27505]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

Focus states for input chips have insufficient contrast.

<img width="788" alt="Screenshot 2023-07-24 at 4 10 58 PM" src="https://github.com/angular/components/assets/7720245/4b8d30ba-48a1-40d8-96b2-e7379c6331d1">
<img width="968" alt="Screenshot 2023-07-24 at 4 13 01 PM" src="https://github.com/angular/components/assets/7720245/b5982835-fd45-4ec8-b24f-0dd901e1c816">

### Reproduction

Steps to reproduce:
1. Navigate to _Chips with input_ demo on https://material.angular.io/components/chips/overview#chips-autocomplete
2. ("Favorite Fruits" appears with "lemon", "lime", "etc")
3. Tab into the "Favorite Fruits" input

### Expected Behavior

"Favor Fruits" input is focused and cursor appears. Contrast ratio of label expected to be 4.5:1 or highter

### Actual Behavior

Cursor appears to be focus, but label contrast ratio is 3.92:1. See screenshots.

### Environment

N/A

**Comment by zarend** [comment]
Action item is to set opacity to 0 on the .mat-mdc-form-field-focus-overlay for input chips. That will make the text easier to read and align closer to Material Design specification.

**Comment by DBowen33** [comment]
![image](https://github.com/angular/components/assets/42016383/b51c4a2d-de01-44ab-9e08-3d6b7c7e7c39)

As of version 17.3.1 mat-label is passing color contrast requirements

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
